scripts/run-pep8.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-MAX_PEP8_VIOLATIONS=113
+MAX_PEP8_VIOLATIONS=110
```

Comment:
While running tests, I noticed that we only had 110 pep8 violations, so I lowered this.